core/src/main/java/com/linecorp/armeria/client/HttpClientBuilder.java

```diff
@@ -62,9 +81,12 @@ public HttpClientBuilder(String uri) {
      *                                  in {@link SessionProtocol}
      */
     public HttpClientBuilder(URI uri) {
```

Comment:
Because we have `HttpClient.of()` which does not take the `uri` as an argument, don't we need to add the same constructor `public HttpClientBuilder() {...}`?

Reply:
Nice idea. :-)

Reply:
fixed :-)